src/olympia/abuse/actions.py

```diff
@@ -324,25 +327,106 @@ def get_owners(self):
 class ContentActionRejectVersion(ContentActionDisableAddon):
     description = 'Add-on version(s) have been rejected'
 
-    def should_hold_action(self):
-        # This action should only be used by reviewer tools, not cinder webhook
-        # eventually, if add-on becomes non-public do as disable
-        raise NotImplementedError
+    def __init__(self, decision):
+        super().__init__(decision)
+        self.content_review = decision.metadata.get('content_review', False)
```

Comment:
fwiw, I did consider using separate Action classes ("ContentActionRejectContent" and "ContentActionRejectContentDelayed"), but it's the same actual outcome, with the emails sent in either case so didn't seem worth the extra complexity - especially as content reviews are being changed soon(?).  

At that point we may want a different ContentAction class - and perhaps set AMO_DISABLE_ADDON instead as the cinder action as it's more what the outcome is (the listing being removed) than a version rejection(?)